picom.sample.conf

```diff
@@ -1,3 +1,28 @@
+################################
```

Comment:
There is not enough explanation here for:
* what does transition do? i.e. what is being transitioned?
* what is transition-offset?
* what is the meaning of each transition-direction?